migration/iD11/oracle/202312080607_IDEMPIERE-5932.sql

```diff
@@ -0,0 +1,206 @@
+;
+
+-- Dec 8, 2023, 6:11:52 AM CST
+UPDATE AD_Window SET Name='Warehouse and Locators',Updated=TO_TIMESTAMP('2023-12-08 06:11:52','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=139
+;
+
+-- Dec 8, 2023, 6:11:52 AM CST
+UPDATE AD_Menu SET Name='Warehouse and Locators', Description='Maintain Warehouses and Locators', Updated=TO_TIMESTAMP('2023-12-08 06:11:52','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=125
+;
+
+-- Dec 8, 2023, 6:11:52 AM CST
+UPDATE AD_WF_Node SET Name='Warehouse and Locators', Description='Maintain Warehouses and Locators', Help='The Warehouse and Locators Window defines each Warehouse, any Locators for that Warehouse and the Accounting parameters to be used for inventory in that Warehouse.',Updated=TO_TIMESTAMP('2023-12-08 06:11:52','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_WF_Node_ID=158
+;
+
+-- Dec 8, 2023, 6:11:57 AM CST
+UPDATE AD_Window SET Name='Window, Tab and Field',Updated=TO_TIMESTAMP('2023-12-08 06:11:57','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=102
+;
+
+-- Dec 8, 2023, 6:11:57 AM CST
+UPDATE AD_Menu SET Name='Window, Tab and Field', Description='Maintain Windows, Tabs & Fields', Updated=TO_TIMESTAMP('2023-12-08 06:11:57','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=141
+;
+
+-- Dec 8, 2023, 6:12:46 AM CST
+UPDATE AD_Window SET Name='Inventory Decrease/Increase',Updated=TO_TIMESTAMP('2023-12-08 06:12:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=341
+;
+
+-- Dec 8, 2023, 6:12:46 AM CST
+UPDATE AD_Menu SET Name='Inventory Decrease/Increase', Description='Enter Internal Use of Inventory', Updated=TO_TIMESTAMP('2023-12-08 06:12:46','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=503
+;
+
+-- Dec 8, 2023, 6:13:01 AM CST
+UPDATE AD_Window SET Name='Payment and Reciept',Updated=TO_TIMESTAMP('2023-12-08 06:13:01','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=195
+;
+
+-- Dec 8, 2023, 6:13:01 AM CST
+UPDATE AD_Menu SET Name='Payment and Reciept', Description='Process Payments and Receipts', Updated=TO_TIMESTAMP('2023-12-08 06:13:01','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=235
+;
+
+-- Dec 8, 2023, 6:15:33 AM CST
+UPDATE AD_Window SET Name='Sales Invoice and Note',Updated=TO_TIMESTAMP('2023-12-08 06:15:33','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=167
+;
+
+-- Dec 8, 2023, 6:15:33 AM CST
+UPDATE AD_Menu SET Name='Sales Invoice and Note', Description='Customer Invoice Entry', Updated=TO_TIMESTAMP('2023-12-08 06:15:33','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=178
+;
+
+-- Dec 8, 2023, 6:15:56 AM CST
+UPDATE AD_Window SET Name='Purchase Invoice and Note',Updated=TO_TIMESTAMP('2023-12-08 06:15:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Window_ID=183
+;
+
+-- Dec 8, 2023, 6:15:56 AM CST
+UPDATE AD_Menu SET Name='Purchase Invoice and Note', Description='Vendor Invoice Entry', Updated=TO_TIMESTAMP('2023-12-08 06:15:56','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Menu_ID=206
```

Comment:
Description is not in sync. Also, perhaps clearer to use Purchase Invoice and Credit/Debit Note instead ?